test/src/main/java/org/apache/accumulo/harness/SharedMiniClusterBase.java

```diff
@@ -124,7 +124,7 @@ private static String getTestClassName() {
     Optional<String> callerClassName =
         StackWalker.getInstance(RETAIN_CLASS_REFERENCE).walk(findCallerITClass).map(Class::getName);
     // use the calling class name, or default to a unique name if IT class can't be found
-    return callerClassName.orElse(String.format("UnknownITClass-{}-{}", System.currentTimeMillis(),
+    return callerClassName.orElse(String.format("UnknownITClass-%d-%s", System.currentTimeMillis(),
```

Comment:
Should both of these be `%d`?

```suggestion
    return callerClassName.orElse(String.format("UnknownITClass-%d-%d", System.currentTimeMillis(),
```

Reply:
@EdColeman Just pinging you for this one, because I'm not sure if you saw it. My guess is that it probably works fine either way.

Reply:
I saw this too but it does work either way since `%s` accepts any type even though we are expecting a long.  Ran some tests in jshell to verify this.

Reply:
I did miss it - corrected in #2390.